lib/jekyll/plugin.rb

```diff
@@ -60,10 +60,7 @@ def self.priority(priority = nil)
     #
     # Returns the safety Boolean.
     def self.safe(safe = nil)
-      if safe
-        @safe = safe
-      end
-      @safe || false
+      @safe ||= !!safe
```

Comment:
This changes the behaviour. Before: If `safe` is truthy, set its value to `@safe`. After: set `@safe` to boolean equivalent of truthiness of `safe` only if `@safe` is not set.

This is a breaking change.